app/src/main/java/org/gdg/frisbee/android/arrow/ArrowActivity.java

```diff
@@ -249,46 +299,108 @@ private void score(final String id) {
             return;
         }
 
-        AppStateManager.load(getGoogleApiClient(), Const.ARROW_DONE_STATE_KEY).setResultCallback(new ResultCallback<AppStateManager.StateResult>() {
-            @Override
-            public void onResult(AppStateManager.StateResult stateResult) {
-                AppStateManager.StateConflictResult conflictResult = stateResult.getConflictResult();
-                AppStateManager.StateLoadedResult loadedResult = stateResult.getLoadedResult();
+        storeInAppState(id);
```

Comment:
Do you think that this is working? I thought AppState API is readonly now. It may not store the data here. Actually if the migration code here is working well, only storing it to the snapshot would be sufficient.